runtime/runtime/src/actions.rs

```diff
@@ -99,6 +99,8 @@ pub(crate) fn action_function_call(
         action_receipt.gas_price,
         action_hash,
     );
+    let output_data_receivers: Vec<_> =
```

Comment:
We should only use this for the final action. For non-final action (in the middle of the batch) should be empty.

Reply:
For final action or final function call in batch?

Reply:
Discussed offline. Final action.